pmsoService/product_flow/features/deliverytrip/app/validators.py

```
@@ -1,3 +1,269 @@
+
+    Args:
+        status: The delivery status to validate
+
+    Raises:
+        ValueError: If validation fails
+    """
+    valid_statuses = {"PENDING", "IN_TRANSIT", "DELIVERED", "CANCELLED", "RETURNED"}
+    if status not in valid_statuses:
+        raise ValueError(
+            f"Trạng thái giao hàng không hợp lệ '{status}'. Phải là một trong: {valid_statuses}"
+        )
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)